flink-connector-jdbc/src/main/java/org/apache/flink/connector/jdbc/table/JdbcRowDataLookupFunction.java

```diff
@@ -116,6 +124,15 @@ public void open(FunctionContext context) throws Exception {
         }
     }
 
+    private FieldNamedPreparedStatement setPredicateParams(FieldNamedPreparedStatement statement)
+            throws SQLException {
+        for (int i = 0; i < pushdownParams.length; ++i) {
```

Comment:
It seems that currently so far there no test checking what happens if we enter inside this loop

Reply:
@snuyanzin Thanks I am working on this currently. I can test the method but am not sure how to check it worked. My current test shows that the sqltext in the statement is not updated with the parameters. I am continuing to look into this.

Reply:
@snuyanzin I am not finding a way to scaffold a unit test around this method and prove the cases work. It looks like I need to prepare the statement and then populate rowdata and then run through the converter before I can setup the statement required for this method. I will continue to investigate.

Reply:
I have added more  tests to the eval method which does the scaffolding already. I hope this is sufficient.